petab/models/sbml_model.py

```diff
@@ -103,8 +104,31 @@ def get_free_parameter_ids_with_values(
             ar.getVariable() for ar in self.sbml_model.getListOfRules()
         }
 
+        parser_settings = libsbml.L3ParserSettings(
+            self.sbml_model,
+            libsbml.L3P_PARSE_LOG_AS_LOG10,
```

Comment:
This is pretty unintuitive, to me it looks like this will convert `log` to `log10`, and I'm not sure if the libsbml docs make sense. e.g. this for `L3P_PARSE_LOG_AS_ERROR`
> Collapse unary minuses where possible when parsing text-string formulas.

https://synonym.caltech.edu/software/libsbml/5.18.0/docs/formatted/python-api/namespacelibsbml.html#a9ef0398726fb45bd1cb24c4a5ed671f0

The docstring for `L3P_PARSE_LOG_AS_LOG10` looks like it actually parses `log` as `log`, not parsing `log` as `log10`, but I'm not sure I trust the docs given the docstring for `L3P_PARSE_LOG_AS_ERROR`. Should we instead use `L3P_PARSE_LOG_AS_LN`?

I think most users would expect `log` to be `ln`, and that `log10` is only when explicitly specified with `log(10, x)` or `log10(x)`.

Reply:
Those are the libsbml default settings. The only relevant thing here is ignoring units.

Reply:
OK, if it actually keeps `log` as `log`, not `log10`, then no problem

Reply:
Note that this is about parsing MathML not, any observableFormula or the like. In MathML `log` without explicit base [means `log10`](https://www.w3.org/TR/mathml4/#contm_log_sec).

Reply:
Oh! Good to know that `log` means different things in PEtab and the model :grimacing: Then I think we could be more careful about a math spec in the next version of PEtab, I added a comment to the draft timecourse spec.